### DISCUSSION TITLE
Request to assign an issue to start contributing

### DISCUSSION THREAD

**Discussion by kishorepran** [lkzhao/Hero#752] [Q&A] [answered]
Hello @JoeMatt I would love to contribute to this project.
Can you please assign me any issues from the list which would add most value to project.
I have read and will follow the contributing guidelines of this project.
Thanks

**Comment by JoeMatt** [comment, marked as answer]
Glad to have a volunteer!

Only issue is since I don't use Hero in any projects myself at the moment, I'm not really sure what's most critical.

There are the issues marked Good First Issue or Needs Help perhaps.

If you have anything specific you are good at, like animations or uiview hacking, crash debugging etc let me know.

**Reply by kishorepran** [reply]
Hello @JoeMatt : Thanks for the update.
Will look into the Good first issue and Need Help first. Will tag you the one I would like to pick up. 
This way I am not working on anything duplicate.
I am good at Crash Debugging not great in animations but more then willing to learn and become equally good at it as well.

**Reply by JoeMatt** [reply]
maybe this then,
https://github.com/HeroTransitions/Hero/issues/734
